web/src/components/molecules/Content/Table/index.tsx

```diff
@@ -509,15 +566,28 @@ const ContentTable: React.FC<Props> = ({
           dataSource={contentTableFields}
           tableAlertOptionRender={AlertOptions}
           rowSelection={rowSelection}
-          columns={[...actionsColumn, ...contentTableColumns]}
+          columns={[...actionsColumn, ...contentColumn]}
```

Comment:
This line is giving an error because the contentColumn might be undefined.
add something like this:
```
const tableColumns = useMemo(() => {
    const items = [...actionsColumn];
    if (contentColumn) items.push(...contentColumn);
    return items;
  }, [actionsColumn, contentColumn]);
  ```